hclpack/structure_test.go

```diff
@@ -85,7 +152,13 @@ func TestBodyContent(t *testing.T) {
 			}
 
 			if !cmp.Equal(test.Want, got) {
-				t.Errorf("wrong result\n%s", cmp.Diff(test.Want, got))
+				bytesAsString := func(s []byte) string {
```

Comment:
This is not directly relevant to this PR but makes it a bit easier to see what's failing the the test.

Copied from https://github.com/hashicorp/hcl2/blob/master/hclpack/pack_native_test.go#L154